src/clj/game/cards/icebreakers.clj

```diff
@@ -706,6 +706,21 @@
                                  (break-sub 1 1 "Code Gate")
                                  (strength-pump 1 1)]})
 
+   "Ika"
+   (auto-icebreaker ["Sentry"]
+                    {:abilities [(break-sub 1 2 "Sentry")
+                                 (strength-pump 2 3 :all-run)
+                                 {:label "2 [Credits]: Host Ika on a piece of ICE"
+                                  :effect (req (let [ika (get-card state card)]
+                                                 (resolve-ability state side
```

Comment:
Can this effect be merged into the main ability? Is the resolve-ability necessary?

Reply:
No. The reason I had this was because I started trying to verify that host and target ice were not the same, but this is unneccessary. Removing it.